platform/configure/single-sign-on/overview.mdx

```diff
@@ -16,16 +16,19 @@ import PartialAuthOther from "../../_partials/auth/sso/generic.mdx";
 
 import PartialMultipleSSO from "../../_partials/install/multiple-sso.mdx";
 
-vCluster Platform supports all major Single-Sign-On (SSO) providers. To configure SSO for vCluster Platform, expand the
+The platform supports all major Single-Sign-On (SSO) providers. To configure SSO
```

Comment:
Let's add in some content that describes how this section assumes you've started the platform and then will be enabling SSO.